sync.sh

```diff
@@ -35,7 +35,15 @@ fi
 
 function sync_data {
     src_data_path="s3://${SRC_DATA_BUCKET}/${SRC_DATA_PREFIX}"
-    
+
+    # Use only the most recent data
```

Comment:
Pulling down multiple days worth of data difficult, the interface may look something like this in the future:

```
integration sync all
integration sync head [-n <INT>]
integration sync date <ds_nodash>
```
Until then, synchronization and reporting will only be able to use the currently visible data (which may not necessarily be a bad thing).